sklearn/linear_model/tests/test_quantile.py

```diff
@@ -242,3 +244,25 @@ def test_linprog_failure():
     msg = "Linear programming for QuantileRegressor did not succeed."
     with pytest.warns(ConvergenceWarning, match=msg):
         reg.fit(X, y)
+
+
+def test_sparse_input():
```

Comment:
Please parametrize this test to test both `fit_intercept=True` and `fit_intercept=False`.

Reply:
We should also parametrize the sparse format in `csr`, `coo`, `csc`.